app/compliance.go

```diff
@@ -16,19 +16,25 @@ func (a *App) GetComplianceReports(page, perPage int) (model.Compliances, *model
 		return nil, model.NewAppError("GetComplianceReports", "ent.compliance.licence_disable.app_error", nil, "", http.StatusNotImplemented)
 	}
 
-	return a.Srv().Store.Compliance().GetAll(page*perPage, perPage)
+	compliance, err := a.Srv().Store.Compliance().GetAll(page*perPage, perPage)
+
```

Comment:
this empty lines looks weird to me, normally that is not the style that we use here. between the execution and the error handling we don't put blank lines.